node/subsystem-util/src/lib.rs

```diff
@@ -828,7 +828,7 @@ where
 		ctx: &mut Context,
 	) -> SubsystemResult<()> {
 		match outgoing.expect("the Jobs stream never ends; qed") {
-			FromJobCommand::SendMessage(msg) => ctx.send_message(msg).await,
+			FromJobCommand::SendMessage(msg) => { let _ = ctx.send_message(msg).await; }
```

Comment:
The problem I see here is that the job will not be aware that some message was dropped.